oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexTest.java

```diff
@@ -980,26 +969,6 @@ private void assertQuery(IndexTracker tracker, NodeState indexed, String key, St
         return paths;
     }
 
-    private static List<String> assertFilter(Filter filter, AdvancedQueryIndex queryIndex,
```

Comment:
Was this removal by mistake ? I see it's still called in this test class.

Reply:
This one is not used at all. The one in use does not specify the `ordered` boolean parameter (see line 954).

Reply:
ah ..ok now I see (I was checking in trunk where it is called from customScoreQuery(), but even that method itself is removed in your changes.)